api/v1alpha4/condition_consts.go

```diff
@@ -127,8 +127,14 @@ const (
 	// VolumeDetachSucceededCondition reports a machine waiting for volumes to be detached.
 	VolumeDetachSucceededCondition ConditionType = "VolumeDetachSucceeded"
 
-	// WaitingForVolumeDetachReason (Severity=Info) provide evidence that a machine node waiting for volumes to be attached.
+	// VolumeDetachTimedOutCondition reports a machine waiting for node volumes detachment timeout to be exceeded.
+	VolumeDetachTimedOutCondition ConditionType = "VolumeDetachTimedOut"
```

Comment:
We shouldn't add the constant to v1alpha4 API 

(applies also to changes in machine_types.go, v1alpha4 API should stay the same, only conversion should be changed)

Reply:
I got rid of all the unneeded changes both v1a3&v1a4 API leaving only conversions, PTAL once again.